gcs/constraints/smart_table.cc

```diff
@@ -733,9 +733,11 @@ auto SmartTable::install(Propagators & propagators, State & initial_state) && ->
 
                         auto flag = unary_set_entry.constraint_type == SmartEntryConstraint::In ? propagators.create_proof_flag("inset") : propagators.create_proof_flag("notinset");
 
-                        propagators.define(initial_state, move(set_value_sum) >= Integer{static_cast<long long>(set_value_sum.terms.size())},
+                        auto set_rhs = Integer{static_cast<long long>(set_value_sum.terms.size())};
```

Comment:
Oh. That would do it... I wonder whether there's a linting tool or something that can catch that kind of thing.

Reply:
To be fair my IDE was flagging it, I had just overlooked / blanked the warning. Lesson learned

Reply:
Ah, GCC doesn't seem to warn for this yet.